### PR TITLE
Add grave junk and script

### PR DESCRIPTION
**Pull request by FluxBorn**
Added this for the grave pile game in HE fest.

### CODE REVIEW

File: grave-pile.lic [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+junk_list = get_settings.grave_junk.map { |x| /\b#{x}/i }
+
+DRCT.walk_to(10_291)
+loop do
+  break if checkbleeding
+  break if checkhealth < 75
+
+  until checkright || checkleft
+    fput('stand') unless checkstanding
```
Comment (rpherbig) [inline grave-pile.lic]: There's a `fix_standing` helper (https://github.com/rpherbig/dr-scripts/blob/master/common.lic#L223) you might want to use here

File: grave-pile.lic [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    if /injured/ =~ DRC.bput('dig pile', 'roundtime', 'You are too injured', 'Perhaps that would work better')
+      DRCT.walk_to(19_104)
```
Comment (Sarvatt) [inline grave-pile.lic]: DRCT.walk_to(10_341) would be better here

File: grave-pile.lic [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      exit
+    end
+    pause 1
+    waitrt?
+  end
+
+  while thing = DRC.left_hand || DRC.right_hand
+    case thing
+    when *junk_list
+      fput("put #{thing} in bin")
```
Comment (Sarvatt) [inline grave-pile.lic]: `bput("put #{thing} in bin", 'You drop')` instead maybe?

File: grave-pile.lic [hunk 4]
```diff
@@ -0,0 +1,36 @@
+custom_require.call(%w[events common-travel common])
```
Comment (rpherbig) [inline grave-pile.lic]: I'd like to see a documentation header here as in the other scripts

File: grave-pile.lic [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    pause 1
+    waitrt?
+  end
+
+  while thing = DRC.left_hand || DRC.right_hand
+    case thing
+    when *junk_list
+      fput("put #{thing} in bin")
+    else
+      # if thing =~ /ticket/
```
Comment (rpherbig) [inline grave-pile.lic]: Should this code be deleted or do you intend to implement this?

Comment (FluxBorn) [reply grave-pile.lic]: I'll get rid of it, to be honest I jacked this from the darkbox script and hastily converted it. Someone asked to have it so I threw it up here. Give me a bit to clean it up some.

File: grave-pile.lic (hunk 3 above)
Comment (rpherbig) [reply grave-pile.lic]: Does `dispose_trash` do what you need?

**Comment by Sarvatt** [comment]
I would commit to cleaning all this up right away if it's accepted, would be good to have it while HE is still on :)

**Pull request merged**